.github/workflows/test.yml

```
@@ -33,7 +33,7 @@ jobs:
       - uses: actions/setup-go@v5
         if: env.GIT_DIFF
         with:
-          go-version: '1.22'
+          go-version: "1.22"
```

can we use the latest stable version?

```suggestion
          go-version: "stable"
```

---

I think we should test ignite with the version we support in the go.mod.
This makes sure we are still backward compatible.